File: quark/test/emit/print/py/print_.py
from quark_runtime import *

class Test(object):
    def _init(self): pass
    def __init__(self): self._init()
    
    def test(self):
        _println("Hello World!");
    


def main():
    (Test()).test();


if __name__ == "__main__":
    main()

File: quark/test/emit/print/js/print.js
var _qrt = require("datawire-quark-core");

// CLASS Test
function Test() {
    this.__init_fields__();
}
exports.Test = Test;

function Test__init_fields__() {}
Test.prototype.__init_fields__ = Test__init_fields__;

function Test_test() {
    _qrt.print("Hello World!");
}
Test.prototype.test = Test_test;

function main() {
    (new Test()).test();
}
exports.main = main;

main();
